ng2-material/components/data_table/data_table_tr.ts

```diff
@@ -1,17 +1,71 @@
-import {Component, Input, Inject, Optional, forwardRef, ElementRef, AfterContentInit} from "angular2/core";
+import {Component, Input, Inject, forwardRef, ElementRef, AfterContentInit} from "angular2/core";
+import {isPresent} from "angular2/src/facade/lang";
 import {MdDataTable} from "./data_table";
 import {MdCheckbox} from "../checkbox/checkbox";
 import "rxjs/add/operator/map";
 
+
+/**
+ * MdDataTable row
+ */
+export interface ITableRow {
+  selectableValue: string;
+  isActive: boolean;
+}
+
 @Component({
-  selector: 'tr',
+  selector: 'tr[md-data-table-header-row]',
```

Comment:
Is it necessary that this select a `tr` in particular? Totally unthinkable someone would want to use directive on some other piece of markup?